apache/rocketmq/v2/admin.proto

```diff
@@ -38,6 +43,105 @@ message ChangeLogLevelRequest {
 
 message ChangeLogLevelResponse { string remark = 1; }
 
+message DescribeTopicStatusRequest {
+  optional string instance_id = 1;
+  Resource topic = 2;
+}
+
+message DescribeTopicStatusResponse {
+  Status status = 1;
+  MessageType topic_message_type = 2;
+  string description = 3;
+  google.protobuf.Timestamp create_timestamp = 4;
+  map<string, string> tags = 5;
+}
+
+message DescribeTopicSubscriptionRequest {
```

Comment:
Why instance_id is still necessary as "topic" itself is already namespaced?

Reply:
The instance_id is different from the namespace. A instance can have multiple namespace.